### ISSUE TITLE
[BUG] Unknown field "proofOps" / Cors error

### ISSUE THREAD

**Issue by bond-igor** [terra-money/core#516]
**Describe the bug**
If I'm using the latest version of LocalTerra i'm getting error in my cmd shell: 'ERROR: ABCIQuery: error unmarshalling result: unknown field "proofOps" in types.ResponseQuery'
If I'm using --branch v0.4.1 of LocalTerra, I can't use terra js because I'm getting cors error: 'Access to XMLHttpRequest at 'http://localhost:1317/txs/estimate_fee' from origin 'http://localhost:4200' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.'
**To Reproduce**
To reproduce "proofOps" error: try to call some of terracli command, for example: terracli query wasm contract <...>
To reproduce Cors error use createAndSignTx function: 
`const mk = new MnemonicKey();
this.client = new LocalTerra();
const wallet = this.client.wallet(mk);
const send = new MsgSend(
  wallet.key.accAddress,
  <any address>,
  { uluna: 1000 }
);
wallet.createAndSignTx({
  msgs: [send],
  memo: "Hello"
}).then(data => console.log(data));`

So Cors error does not throwed if use the latest branch of LocalTerra, but I can't use my cmd shell in this case

**Comment by YunSuk-Yeo** [comment]
Can you use latest LocalTerra main branch?

**Comment by bond-igor** [comment]
> Can you use latest LocalTerra main branch?

In this case, I'm getting an error: 'ERROR: ABCIQuery: error unmarshalling result: unknown field "proofOps" in types.ResponseQuery'

**Comment by YunSuk-Yeo** [comment]
@bond-igor If you use bombay or v0.5.0-rc0 core, you have to use terrad instead terracli. 

All terracli features are merged into terrad like
```
$ terrad tx ~~
$ terrad query ~~
```
